### ISSUE TITLE
Some texts in the sign in page have very low contrast

### ISSUE THREAD

**Issue by Rakielle** [uyuni-project/uyuni#6776]
# Short description
Some texts in the sign in page of Uyuni have very low contrast. Improving the contrast levels for the texts would improve accessibility.

# Details
As part of openSUSE/mentoring#190, this issue hopes to improve on the accessibility for the sign in page. Currently, there are some texts on the page that have very low contrast, the texts and background colours have contrast ratio that are below the 4.5:1 that should be between the foreground (text) and the background.
<img width="203" alt="Screenshot 2023-03-23 at 22 46 07" src="https://user-images.githubusercontent.com/48498778/227373565-88f375dc-1832-4320-93c3-e60da79b037a.png">
<img width="119" alt="Screenshot 2023-03-23 at 22 45 59" src="https://user-images.githubusercontent.com/48498778/227373810-af80bda8-ef52-4af3-b40c-044be3deb62d.png">
 Adequate contrast of text is necessary for all users, especially users with low vision.

To fix this, we should increase the contrast between the foreground (text) colour and the background colour. This can be done by using a darker shade of the current colours for the texts.

Using `#00D18B` for the `View website` and `#B5B8BA` for the `footer` texts achieves the right contrast to improve the accessibility for the text.

![btn](https://user-images.githubusercontent.com/48498778/227560390-af56d75c-85a6-48bf-ba75-fe1bb86f64d1.png)
![footer](https://user-images.githubusercontent.com/48498778/227560478-aff1647b-a82e-427c-b8ae-8d0765ebc423.png)

# References
[Contrast (Minimum)](https://webaim.org/standards/wcag/checklist#sc1.4.3)

**Comment by Rakielle** [comment]
@cbbayburt @deneb-alpha I opened a PR for this issue here: https://github.com/uyuni-project/uyuni/pull/7051.
PTAL
